app/views/sessions/payment.html.erb

```diff
@@ -0,0 +1,112 @@
+<br/>
+<img src="/assets/logo.png" alt = "Background">
+<br/>
+<br/>
+
```

Comment:
can you add a url at the top from where you got some of this code?

Reply:
I literally got this code from the welcome page you made lol